integration-tests/test_client.py

```diff
@@ -0,0 +1,52 @@
+import contextlib
+from utils import rhcd_service_is_active
+
```

Comment:
I would rename this file to `test_connect.py`, because this repository contains only client.

I can also see that all new tests are just skipped: https://artifacts.dev.testing-farm.io/120598b7-e751-420b-81af-5ea45d2cd2af/ What influences that test is skipped?

Reply:
I would expect that this file also includes doc string containing explanation of its purpose. It would be also good to add reference to https://github.com/ptoscano/pytest-client-tools/, because this Python module is used here.

Reply:
All new tests are getting skipped because we do not yet have configuration/credentials setup upstream. It should run downstream for now.

Reply:
> I would expect that this file also includes doc string containing explanation of its purpose. It would be also good to add reference to https://github.com/ptoscano/pytest-client-tools/, because this Python module is used here.

this repo is in requirements.txt, can you be more specific about how to add reference, as in a comment about it would do? or something else is expected here?

Reply:
Each Python module should contain doc string with short explanation like this file:
https://github.com/candlepin/subscription-manager/blob/main/src/cloud_what/providers/aws.py

Why could it be useful? Imagine that you are community contributor and you want to extend integration tests, because you added new feature. It would be nice to see doc string with short explanation and recommended link, because https://github.com/ptoscano/pytest-client-tools/ contains more information and some documentation.

Something like this:

```
"""
This Python module contains integration tests of rhc. it uses pytest-client-tools Python module.
More information about this module could be found: https://github.com/ptoscano/pytest-client-tools/
"""
```

Reply:
> Each Python module should contain doc string with short explanation like this file: https://github.com/candlepin/subscription-manager/blob/main/src/cloud_what/providers/aws.py
> 
> Why could it be useful? Imagine that you are community contributor and you want to extend integration tests, because you added new feature. It would be nice to see doc string with short explanation and recommended link, because https://github.com/ptoscano/pytest-client-tools/ contains more information and some documentation.
> 
> Something like this:
> 
> ```
> """
> This Python module contains integration tests of rhc. it uses pytest-client-tools Python module.
> More information about this module could be found: https://github.com/ptoscano/pytest-client-tools/
> """
> ```

so this docstring should exist in all test scripts?

Reply:
I'm not telling you to modify all existing test scripts of all our projects. :-) I'm telling you that it is good practice, and if we start adding new tests scripts to this project, then it would be good to add these doc strings to all new `.py` files.

Reply:
Done.